__init__.py

```diff
@@ -8,13 +8,21 @@
     load_single_type, load_single_type_summary,
     install_cache)
 
+import datetime
 import sys
 import locale
+import jinja2
 locale.setlocale(locale.LC_ALL, '')
 
 app = Flask(__name__)
 
+@jinja2.contextfunction
+def get_context(c):
+    return c
+
 app.jinja_env.globals['api_location'] = api_location
+app.jinja_env.globals['context'] = get_context
+app.jinja_env.globals['callable'] = callable
```

Comment:
Sorry, I meant `callable`, not `context`. Is `callable` used in the jinja templates?